source/Application/Component/Widget/ArticleDetails.php

```diff
@@ -646,9 +631,7 @@ public function showZoomPics()
      */
     public function getZoomPics()
     {
-        $aPicGallery = $this->getPictureGallery();
```

Comment:
You can easily checkout `$this->_aPicGallery` or `getPictureGallery`, by "stepping into" this breakpoint.
![image](https://cloud.githubusercontent.com/assets/1001186/16195612/c03d6e5c-36fa-11e6-8b59-1ce27b4ee7fd.png)